03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,18 @@
+Q #1
+A request handler with access to the application's 
+request-response cycle is known as middleware. It's a 
+function that holds the request object, the response object, 
+and the middleware function. Middleware can also send the response 
+to the server before the request.
```

Comment:
A few clarifying questions!

> It's a function that holds the request object, the response object

What do you mean by it, "holding" these objects?

> ... and the middleware function.

The middleware function holds the middleware function?